src/main/java/Config/Routes/OPTIONSRoute.java

```diff
@@ -0,0 +1,12 @@
+package Config.Routes;
+
+import Config.Method;
+import Controller.Handler.Handler;
+
+public class OPTIONSRoute extends Route {
+	public OPTIONSRoute(Method method, String path, Handler handler) {
+		this.method = method;
+		this.path = path;
+		this.handler = handler;
+	}
+}
```

Comment:
So, I noticed that all of these route objects have the same structure.  Is it possible you could just have one structure?  Like so:

```java
// Notice I change the interface from Route to Routed
//
public class Route extends Routed {
	public Route(Method method, String path, Handler handler) {
		this.method = method;
		this.path = path;
		this.handler = handler;
	}
}
```

Or you could have specialized custom versions, like so:

```java
public class OPTIONSRoute extends Route {
	public OPTIONSRoute(String path, Handler handler) {
		this.method = Config.Method.OPTIONS;
		this.path = path;
		this.handler = handler;
	}
}
```